avaframe/out3Plot/outAna1Plots.py

```diff
@@ -232,153 +234,312 @@ def plotErrorTime(time, hErrorL2Array, hErrorLMaxArray, vhErrorL2Array, vhErrorL
     pU.saveAndOrPlot({'pathResult': outDirTest / 'pics'}, 'Error_Time_' + str(outputName), fig1)
 
 
-def plotError(simDF, outDirTest, cfgSimi):
+def plotErrorConvergence(simDF, outDirTest, cfgSimi, xField, yFieldArray, coloredBy, sizedBy, logScale=False, fit=False):
```

Comment:
Please add a few more oneline comments inside this function to explain what is going on